app/components/ContentEditable.tsx

```diff
@@ -129,6 +129,29 @@ const ContentEditable = React.forwardRef(
       []
     );
 
+    if (React.Children.count(children) > 1) {
```

Comment:
Clever, but I'm not a fan of the copy/paste here in `<Content` – probably better to just introduce an optional `beforeTitle` prop or similar.